src/commands/build/wranglerjs/bundle.rs

```diff
@@ -50,8 +52,9 @@ impl Bundle {
 
         script_file.write_all(script.as_bytes())?;
 
+        let metadata = create_metadata(self).expect("could not create metadata");
         let mut metadata_file = File::create(self.metadata_path())?;
-        metadata_file.write_all(create_metadata(self).as_bytes())?;
+        metadata_file.write_all(metadata.as_bytes())?;
```

Comment:
I know this didn't originate from this PR, but why do we write the metadata to a file?

Reply:
I think its useful if you want to use wrangler as a "build-only" tool

Reply:
But metadata isn't used unless you're publishing to Cloudflare,  so we support building with wrangler and then publishing with curl or something?

Reply:
Maybe as part of a CI process where the CI handles upload because wrangler doesn't support complex config for handling staging environments etc. I think it's useful for all "files" that we upload for multipart to end up on disk somewhere for this reason.

Reply:
does it stay on disk? or is it cleaned up later?

Reply:
There was some back and forth with the metadata file lately, see #276. The file is store in ./worker as part of the worker "bundle", it is persistent.

I'm sure some people will commit their worker, it make sense to have a consistent state including the metadata, it's useful for the preview and debugging too.